LICENCE.rst

```diff
@@ -1,8 +1,10 @@
 GrantNav
 ========
 
+Note: This license doesn't apply to the 360Giving theme and related assets (such as the logo).
```

Comment:
I believe that, in the UK at least, look-and-feel falls into trademark territory rather than copyrightability. The code to bring about that look-and-feel can be released under an open license, though - we don't care if someone downloads it, changes some numbers and makes a new blue CoVE. 360 would have something to say if they put something on the Internet that made out that it was related to them - but that would be under passing-off rules rather than copyright. 

Does it make sense for the license to cover all code but not logos, and then to provide a notice that the look-and-feel is that of 360Giving and care should be taken if running public copies of CoVE not to potentially confuse users?